woocommerce/woocommerce
# CYS: disable zoom out on fonts/color pairs iframe

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR disables the zoom-out functionality specifically for the font pair and color palette preview iframes. The zoom-out feature will remain active only for the main site preview.


To maintain a consistent user experience, all preview components (main site, font pair, and color palette) share the same iframe structure. However, the zoom-out functionality should be restricted to the main site preview, as it is unnecessary in the context of the font pair and color palette previews.

| Before | After |
|--------|--------|
| <video src="https://github.com/user-attachments/assets/4b43d72d-f647-4292-962c-73819696c5ac" /> | <video src="https://github.com/user-attachments/assets/860c3515-54c1-4992-be5a-ad1946023fe1" /> | 









### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Head over to WooCommerce > Home > Customize your store.
2. Follow the flow.
3. Click on Choose your fonts
4. On the top menu, click on the lens
5. Ensure that the site preview is zoomed out, but the font pairs preview doesn't change



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: disable zoom out on fonts/color pairs iframe
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
